public/coda/coding_progress.js

```diff
@@ -4,10 +4,10 @@ progress_fs.settings(settings);
 progress_fs.doc('metrics/coda').onSnapshot(res => {
     update_progress_ui(res.data());
 });
-
 function update_progress_ui(data) {
     console.log("update_ui: " + JSON.stringify(data));
     var status_body = document.getElementById('coding_status_body');
+
```

Comment:
I don't think there's a need for this additional whitespace? it doesn't really help reading and results in less code on the screen

Reply:
Great have removed it